styleguide/source/assets/scss/01-atoms/_videos.scss

```diff
@@ -9,3 +9,16 @@ figcaption {
   font-style: italic;
 }
 
+.ama__video-container {
```

Comment:
`container` here should be a sub element of `ama__video` and denoted with a double underscore.

Reply:
great catch